src/modules/getBullsAndCows.js

```diff
@@ -14,6 +14,30 @@
  */
 function getBullsAndCows(userInput, numberToGuess) {
   /* Write your code here */
+  const userInputArr = userInput.toString().split('');
+  const numberToGuessArr = numberToGuess.toString().split('');
+  const result = { bulls: 0, cows: 0 };
+
+  for (let i = 0; i < userInputArr.length; i++) {
+    if (
+      numberToGuessArr.includes(userInputArr[i]) &&
+      numberToGuessArr.indexOf(userInputArr[i]) === i
```

Comment:
The condition for checking bulls is incorrect. It should check if the digits are the same and in the same position, but the current logic checks if the digit exists in the array and is in the same position, which is redundant. Consider simplifying this condition to directly compare the digits at the same index.